src/main/java/ch/njol/skript/expressions/ExprItems.java

```diff
@@ -27,7 +27,7 @@
 @Name("Items")
 @Description("Items or blocks of a specific type, useful for looping.")
 @Examples({
-		"loop items of type ore and log:",
+		"loop tag values of minecraft tag \"diamond_ores\" and \"oak_logs\":",
```

Comment:
you can't get the values of 2 tags like this currently. It'd have to be `tag values of tag "diamond_ores" and tag values of tag "oak_logs"`